platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -754,14 +754,25 @@ public void addOnDidFinishRenderingMapListener(OnDidFinishRenderingMapListener l
   }
 
   /**
-   * Remove a callback that's invoked when the map has finished rendering.
+   * Set a callback that's invoked when the map has entered the idle state.
    *
-   * @param listener The callback that's invoked when the map has finished rendering
+   * @param listener The callback that's invoked when the map has entered the idle state.
+   */
+  public void addOnDidEnterIdleListener(OnDidEnterIdleListener listener) {
+    mapChangeReceiver.addOnDidEnterIdleListener(listener);
+  }
+
+  /**
+   * Remove a callback that's invoked when the map has entered the idle state.
+   *
+   * @param listener The callback that's invoked when the map has entered the idle state.
    */
-  public void removeOnDidFinishRenderingMapListener(OnDidFinishRenderingMapListener listener) {
-    mapChangeReceiver.removeOnDidFinishRenderingMapListener(listener);
+  public void removeOnDidEnterIdleListener(OnDidEnterIdleListener listener) {
+    mapChangeReceiver.removeOnDidEnterIdleListener(listener);
```

Comment:
We are removing `MapView#removeOnDidFinishRenderingMapListener `. Seeing the other changes in this PR I think this should be kept.

Reply:
Ack, thanks for catching!